verl/workers/fsdp_workers.py

```diff
@@ -119,6 +120,10 @@ def _build_model_optimizer(self,
         # override model kwargs
         actor_model_config = AutoConfig.from_pretrained(local_path, trust_remote_code=trust_remote_code)
 
+        if use_rmpad:
+            from verl.models.registry import if_support_rmpad
```

Comment:
The name if_support_rmpad is mislead, how about check_model_support_rmpad?

Reply:
Sure